network-grpc/src/lib.rs

```diff
@@ -16,3 +16,5 @@ pub mod client;
 mod convert;
 pub mod server;
 mod service;
+
+pub const PROTOCOL_VERSION: u32 = 0;
```

Comment:
I would not use `0` as an initial version and keep `0` for error/garbage cases

Reply:
Now that you mention it, 0 is the no-field default in protobuf 3.
I will change it to 1, but we'll keep breaking the protocol without bumping the version until we declare it stable.